src/nft/components/details/AssetDetails.css.ts

```diff
@@ -23,6 +23,42 @@ export const image = style([
   },
 ])
 
+export const embedContainer = style([
+  {
+    position: 'relative',
+    overflow: 'hidden',
+    width: '100%',
+    paddingTop: '100%',
+  },
+])
+
+export const embed = style([
+  sprinkles({ borderRadius: '20', height: 'full', alignSelf: 'center' }),
+  {
+    maxHeight: 'calc(90vh - 165px)',
+    minHeight: 400,
+    maxWidth: 780,
+    boxShadow: `0px 20px 50px var(--shadow), 0px 10px 50px rgba(70, 115, 250, 0.2)`,
+    '@media': {
+      '(max-width: 1024px)': {
+        maxHeight: '64vh',
+      },
+      '(max-width: 640px)': {
+        minHeight: 280,
+        maxHeight: '56vh',
+        maxWidth: '100%',
+      },
+    },
+    position: 'absolute',
+    top: 0,
+    left: 0,
+    bottom: 0,
+    right: 0,
+    width: '100%',
+    height: '100%',
+  },
+])
+
```

Comment:
We're actually trying to move away from sprinkles and start using `styled-components` wherever possible. (Sorry, I know there's still a lot of sprinkles used in this area of the codebase!)

Reply:
@JFrankfurt sounds good, will update!